sdk/identity/identity/test/public/node/environmentCredential.spec.ts

```diff
@@ -4,7 +4,7 @@
 /* eslint-disable @typescript-eslint/no-non-null-asserted-optional-chain */
 
 import sinon from "sinon";
-import { assert } from "chai";
+import assert from "assert";
```

Comment:
Just an extra reminder that we don’t want to do this :)

Reply:
ooh I was getting error :(